theia-extensions/theia-blueprint-updater/src/electron-browser/updater/theia-updater-frontend-contribution.ts

```diff
@@ -204,8 +206,19 @@ export class TheiaUpdaterFrontendContribution implements CommandContribution, Me
         }
     }
 
-    protected async handleError(error: string): Promise<void> {
-        this.messageService.error(error);
+    protected async handleError(error: UpdaterError): Promise<void> {
+        if (error.errorLogPath) {
+            const answer = await this.messageService.error(error.message, 'View Error Log');
```

Comment:
nitpick: you could refactor `View Error Log` to a constant since it's being used twice but I'm fine with this approach as well :+1:

Reply:
I moved it to a constant.